### ISSUE TITLE
[docs] Docs missing online for nn.functional.normalize

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#1946]
Docs are [written](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L954-L973) but not [visible](http://pytorch.org/docs/master/nn.html#torch-nn-functional).

**Comment by soumith** [comment]
fixed via https://github.com/pytorch/pytorch/commit/37183e91de5593922f835f411670fbc96a739012 (will regenerate website in about 10 mins)
